docs/operating-system/getting-started.md

```diff
@@ -82,20 +80,17 @@ To build for multiple targets in a single source directory, separate output dire
 
 
 ```shell
-sudo scripts/enter.sh make O=output_rpi4_64 rpi4_64
+scripts/enter.sh make O=output_rpi4_64 rpi4_64
 ```
 
 ### Use the build container interactively
 
 If no argument to `scripts/enter.sh` is passed, a shell will be presented.
 
 ```bash
-$ sudo scripts/enter.sh
-Sending build context to Docker daemon  159.7kB
-Step 1/8 : FROM debian:bullseye
- ---> a178460bae57
+$ scripts/enter.sh
 [...]
-builder@c6dfb4cd4036:/build$ 
+builder@d3d7577663c9:/build$
```

Comment:
Just noticed this quite a pointless change as it's always random, I just blindly copied and cropped the output. But does no harm.